aml-development/ozp-center
# feat(notifications, quickview, profiles): Additional load indicators

Add load indicators to various areas of the application, specifically: loading user profiles (listings), tabs in the listing quickview (administration, reviews, notifications), center settings (notifications)

closes AMLNG-808, AMLNG-809, relates to AMLNG-838

**Description**
When loading the user profile, listing quickview, and loading notifications in center settings, update loading process to implement indicator used when Center page first loads.

**Acceptance Criteria**
use site data loading indicator image as placeholder until result set is displayed
same layout as Center page

**How to test code**
Pull additional_load_indicators from ozp-react-commons
Pull additional_load_indicators from ozp-center
Ensure that ozp-center is built with the additional_load_indicators branch of ozp-react-commons
